engines/python/setup/djl_python/chat_completions/chat_properties.py

```diff
@@ -10,190 +10,15 @@
 # or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS"
 # BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied. See the License for
 # the specific language governing permissions and limitations under the License.
-from collections import defaultdict
-from typing import Optional, Union, List, Dict, Any, Tuple
-from pydantic import BaseModel, Field, field_validator, ValidationInfo, ConfigDict
-from PIL.Image import Image
+from typing import Optional
+from pydantic import Field
+from vllm.entrypoints.openai.protocol import ChatCompletionRequest
```

Comment:
This should be ok in the LMI and Neuron containers (we should validate on the neuron side that this does work), but I don't know if it will work in the trtllm container since we don't install vllm there. 

We should either install vllm in the trtllm container, or retain the old messages format for trtllm.

Reply:
After some testing, I think we need to retain the old format. Because [parse_chat_messages()](https://github.com/vllm-project/vllm/blob/main/vllm/entrypoints/chat_utils.py#L912) need a ModelConfig object. We can't get/create ModelConfig object in trtllm container.